frontend/src/app/components/onboarding/onboarding.component.html

```diff
@@ -5,14 +5,17 @@ <h1 data-cy="titel">Puzzle ScrumPoker</h1>
 <p>Username erstellen</p>
 <div>
   <label for="fname">Username: </label>
-  <input #un id="fname" name="fname" placeholder="Username" type="text" data-cy="username">
+  <input [(ngModel)]="username" id="fname" name="fname" placeholder="Username" type="text" data-cy="username">
 </div>
 <div>
   <p>Dies ist dein Benutzername, welcher von allen anderen Spieler zu sehen sein wird. Verwende keine Sonderzeichen.</p>
 </div>
-<br>
 <div>
-  <button [disabled]="clicked" (click)="create(un.value)" data-cy="onboarding-button">Erstellen</button>
+  <button [disabled]="this.checkUsername(username)" (click)="create(username)" [class.border-red]="this.checkUsername(username)" data-cy="onboarding-button">Erstellen</button>
+  <div *ngIf="checkUsername(username); let result">
+    {{result}}
```

Comment:
Die Methode checkUsername(...) wird dreimal aufgerufen:
Überprüfen, ob man um das ganze Template nicht ein Element wie <ng-container>, <ng-content> oder <ng-template> erstellen kann, welches die Methode checkUsername(...) aufruft und das Ergebnis in einer Variable speichert. Ähnlich, wie wir es beim *ngIf gemacht haben.